Components/HeroPostHeader/Partials/PublishDate/index.twig

```diff
@@ -0,0 +1,3 @@
+<div class="publishDate">
+  Published at {{ post.date }}
```

Comment:
Why is this hard coded text? You could make a field + `str_replace` to add the date dynamically